tests/test_ma_functions.py

```diff
@@ -0,0 +1,59 @@
+from unittest import TestCase
+
+import numpy.ma as ma
+from numpy.ma.testutils import assert_equal
+
+from foapy.ma.alphabet import alphabet
+
+
+class TestMaAlphabet(TestCase):
+    """
+    Test list of unique masked_array elements
+    """
+
+    def test_string_values_with_mask(self):
+        assert_equal(
+            alphabet(
+                ma.masked_array(["a", "c", "c", "e", "d", "a"], [0, 0, 0, 1, 0, 0])
+            ),
+            ma.masked_array(["a", "c", "e", "d"], mask=[0, 0, 1, 0]),
+        )
```

Comment:
```suggestion
        X=ma.masked_array(["a", "c", "c", "e", "d", "a"], [0, 0, 0, 1, 0, 0])
        
        expected=ma.masked_array(["a", "c", "e", "d"], mask=[0, 0, 1, 0])
        
        exists=alphabet(X)
        
        assert_equal(expected, exists)
```

pls redo all tests according to this structure ^